drop-transfer/src/ws/events.rs

```diff
@@ -140,14 +147,15 @@ impl<T: Transfer> FileEventTx<T> {
         lock.emit_event(event);
     }
 
-    async fn force_stop(&self, event: Event, status: Result<(), i32>) {
+    async fn terminal(&self, event: Event, status: Result<(), i32>) {
```

Comment:
Id maybe call this `terminate` to better convey what it does
In general curious why the rename?

Reply:
`terminal` is the term we use to describe the state so it better conveys the message about what is about to happen. `force_stop` wasn't bad but wasn't as readable :) IMO

Reply:
in my mind if we already have a state `Terminal`, and we have a function that does the transition to said state, it makes sense to call it `terminate`, but this is an extremely nitpick-y and pedantic view, so feel free to just resolve :)

Reply:
I think I'll change the name to `terminate` since it's a verb and this is the function name so it makes sense for it to be a verb

Reply:
Done